samples/back_stack/src/main/java/com/badoo/ribs/samples/back_stack/parent/ParentPresenter.kt

```diff
@@ -0,0 +1,88 @@
+package com.badoo.ribs.samples.back_stack.parent
+
+import androidx.lifecycle.Lifecycle
+import com.badoo.ribs.android.subscribe
+import com.badoo.ribs.core.plugin.ViewAware
+import com.badoo.ribs.routing.source.backstack.BackStack
+import com.badoo.ribs.routing.source.backstack.operation.newRoot
+import com.badoo.ribs.routing.source.backstack.operation.push
+import com.badoo.ribs.routing.source.backstack.operation.pushOverlay
+import com.badoo.ribs.routing.source.backstack.operation.replace
+import com.badoo.ribs.routing.source.backstack.operation.singleTop
+import com.badoo.ribs.samples.back_stack.parent.routing.ParentRouter
+
+typealias ViewEventContent = ParentView.Event.RoutingContent.Content
+typealias ViewEventOverlay = ParentView.Event.RoutingOverlay.Overlay
+
+typealias RouterConfigurationContentA = ParentRouter.Configuration.Content.A
+typealias RouterConfigurationContentB = ParentRouter.Configuration.Content.B
+typealias RouterConfigurationContentC = ParentRouter.Configuration.Content.C
+typealias RouterConfigurationContentD = ParentRouter.Configuration.Content.D
+typealias RouterConfigurationOverlayE = ParentRouter.Configuration.Overlay.E
+typealias RouterConfigurationOverlayF = ParentRouter.Configuration.Overlay.F
```

Comment:
Let's import Content and Overlay instead of long typealiases. Then you can refer to them as `Content.A`.

```suggestion
import  com.badoo.ribs.samples.back_stack.parent.routing.ParentRouter.Configuration.Content
import  com.badoo.ribs.samples.back_stack.parent.routing.ParentRouter.Configuration.Overlay
```

Reply:
Will do! At beginning I didn't want to refer to them just as `Content.A` to avoid confusion with the enum one in `ParentView.Event`